pkg/instrumentors/bpf/github.com/gorilla/mux/probe.go

```diff
@@ -112,41 +112,51 @@ func (g *Instrumentor) Load(ctx *context.InstrumentorContext) error {
 		return err
 	}
 
-	offset, err := ctx.TargetDetails.GetFunctionOffset(g.FuncNames()[0])
+	for _, funcName := range g.FuncNames() {
+		g.registerProbes(ctx, funcName)
+	}
+	rd, err := perf.NewReader(g.bpfObjects.Events, os.Getpagesize())
 	if err != nil {
 		return err
 	}
+	g.eventsReader = rd
+
+	return nil
+}
+
+func (g *Instrumentor) registerProbes(ctx *context.InstrumentorContext, funcName string) {
```

Comment:
This isn't strictly needed as the probe only listens to one funcName, however it does make it consistent with the other HTTP probes so will make it easier to combine them when we come to it (see #101).